apps/website/src/server/routers/jobs.ts

```diff
@@ -0,0 +1,22 @@
+import { jobPostingTable } from '@bluedot/db';
+import { publicProcedure, router } from '../trpc';
+import db from '../../lib/api/db';
+
+export const jobsRouter = router({
+  getJobs: publicProcedure
+    .query(async () => {
+      const allJobs = await db.scan(jobPostingTable, {
+        publicationStatus: 'Published',
+      });
+
+      // Sort jobs alphabetically by title and remove the body field
+      // TODO: just fetch these fields from the DB in the first place
```

Comment:
I'm not sure we actually have a way to only select particular fields using the airtable wrapper? Is the recommended way to directly query the wrapped db?